apply-pr.js

```diff
@@ -37,9 +37,11 @@ var Transform = require('stream').Transform;
 
 var lstream = require('lstream');
 
+var args = /(.*)\/(.*)#(\d*)/i.exec(process.argv[2]);
```

Comment:
We may want to check `process.argv` before executing the regex, or to check the return value (`args`) to avoid exceptions when accessing items in the `args` array, and instead display some help on how to use `git-apply-pr`.